pkgs/tools/typesetting/tex/texlive/tlpdb-overrides.nix

```diff
@@ -126,6 +126,11 @@ in lib.recursiveUpdate orig rec {
     "mtxrun.lua" = tl.context.tex + "/scripts/context/lua/mtxrun.lua";
   };
 
+  context-legacy.binlinks = {
```

Comment:
I'm not sure why, but "ctxtools" and "pstopdf" are not in `context-legacy.binfiles`. I wasn't sure if we are supposed to still provide them or not.